### DISCUSSION TITLE
each agent has different install package?

### DISCUSSION THREAD

**Discussion by peterlymo** [Velocidex/velociraptor#2199] [Q&A]
i have ubuntu 22.x and ubuntu 20.x well i installed agent on 22.x which is also a velaciraptor server, its succesfull but when i try to install on ubuntu 20.x nothing showed

clear me:

same agent package can be used with mutliple computer ?

is client.config.yaml need to be created each time i want to add new agent to computer?

**Comment by scudette** [comment]
What is the error? Is it just failing to install?
